packages/enzyme/src/RSTTraversal.js

```diff
@@ -44,6 +35,31 @@ export function treeFilter(tree, fn) {
   return results;
 }
 
+/**
+ * To support sibling selectors we need to be able to find
+ * the siblings of a node. The easiest way to do that is find
+ * the parent of the node and access its children.
+ * 
+ * This would be unneeded if the RST spec included sibling pointers
+ * such as node.nextSibling and node.prevSibling
+ * @param {*} root 
+ * @param {*} targetNode 
+ */
+export function findParentNode(root, targetNode) {
```

Comment:
just curious.... RSTTraversal.js has a `parentsOfNode` function that you could potentially use for this? Did it not work when you tried or did you not know about it?

Reply:
I did see that, and it wasn't working for me. I didn't spend a lot of time trying to fix it though since I wasn't really following how it was implemented. I'm not actually sure how it's working at all, since it calls `reverse()` on the result of `pathToNode`, but [`pathToNode` returns `null`](https://github.com/airbnb/enzyme/blob/master/packages/enzyme/src/RSTTraversal.js#L71), which is the error I was getting

```error
TypeError: Cannot read property 'reverse' of null
      at parentsOfNode (packages/enzyme/build/RSTTraversal.js:121:10)
```

It's also more complicated than we need for selector matching since AFAICT it finds all parents, not just the direct parent. If we add a `parent` pointer per discussions in https://github.com/airbnb/enzyme/issues/1085 this utility could go away altogether.

Reply:
Note that that error is a pre-existing bug in enzyme 2 - #410